src/NFTWorker.ts

```diff
@@ -172,6 +172,8 @@ export default class NFTWorker {
                                     }
                                 }, 2000)
                             }
+                            const markerReady = new CustomEvent('markerReady', { detail : {url: this.markerURL} })
```

Comment:
But i didn't tested the code, tell me if it make sense.

Reply:
wait.. i saw that in your code.. but how do you catch that event? the uuid should be in the name of the event... but can be added to the "detail"

Reply:
uuid and name are properies of the NFTWorker class:
https://github.com/webarkit/ARnft/blob/aef6b981db664d30a7fcf6e1bfa72848e5b312f2/src/NFTWorker.ts#L50-L51

they are filled with values when you create a new instance of the class with the constructor:
https://github.com/webarkit/ARnft/blob/aef6b981db664d30a7fcf6e1bfa72848e5b312f2/src/NFTWorker.ts#L60-L66

so that code it should works.

Reply:
> yeah right.. but im using a component in vuejs (like react) and I cant add an EventListener to an Event with an "unknown" name.. doesn't sound like the best approach for creating an event. maybe im wrong... but then how do I catch the event on vue?

In this PR https://github.com/webarkit/ARnft/pull/208 i'm adding global namespaces to tthe arNFT object, in this sense will be easier to get some data, and attach the listeners. The problem is many listeners in ARnft and ARnftThreejs are attached to the document object and not the window ( or the global object for a Node.js environnment) so it's a bit tricky works with listeners at the moment.